docs/source/DeveloperGuide/AddingNewComponents/Utilities/gLogger/gLogger/Basics/index.rst

```diff
@@ -441,100 +441,275 @@ Backend presentation
+                <param5> = <value5>
+            }
+            <backendID6>
+            {
+                <param6> = <value6>
+            }
+        }
+    }
 
-+--------+-----------+------------------------------------------------------+----------------------+
-| Type   | Option    | Description                                          | Default value        |
-+========+===========+======================================================+======================+
-| file   | FileName  | name of the file where the log records must be sent  | Dirac-log\_[pid].log |
-+--------+-----------+------------------------------------------------------+----------------------+
-| server | SleepTime | sleep time in seconds                                | 150                  |
-+--------+-----------+------------------------------------------------------+----------------------+
+Moreover, a same *Backend* identifier can be declared in the both sections in order to update it. Indeed, such a declaration triggers a parameters merger. In case of parameters conflicts, the local parameters are always choosen. Here is an example:   
 
-We can also notice that the *server Backend* requires that the
-*Framework/SystemLogging* service is running in order to send log
-records to a log server.
+::
+    
+    <Systems>
+    {
+        Agents
+        {
+            <Agent1>
+            {
+                ...
+                BackendsConfig
+                {
+                    <backendID1>
+                    {                   
+                        <param1> = <value1>
+                        <param2> = <value2>
+                    }
+                }
+            }
+        }
+    }
+    Resources
+    {
+        LogBackends
+        {
+            <backendID1>
+            {
+                Plugin = <backendClass1>
+                <param1> = <value4>
+                <param3> = <value3>
+            }
+        }
+    }
+    
+In this case, *gLogger* in *<Agent1>* will have one *Backend* instance of the *<backendClass1>Backend* class which will have 3 parameters: 
+
++ <param1> = <value1>
++ <param2> = <value2>
++ <param3> = <value3>
+
+Use the *Backend* resources
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Once our *Backend* resources are declared, we have to specify where we want to use them and we have many possibilities. First of all, we can add them for the totality of the components. This can be made in the */Operations/defaults/* section. Here is the way to proceed: 
+
+::
+    
+    Operations
+    {
+        Defaults
+        {
+            defaultBackends = <backendID1>, <backendID2>, <backendID3>
+        }
+    }
+
+We can also add them for a specific component type, the agents or the services for instance. Such a declaration will overwrite the previous one for the component type choosen: 
+
+::
+    
+    Operations
+    {
+        Defaults
+        {
+            default<componentType>sBackends = <backendID1>, <backendID2>, <backendID3>
```

Comment:
As I said earlier, this we do not want. We already have the possibility to set a component's config in the component itself, that's enough